### DISCUSSION TITLE
Need some starting guidance

### DISCUSSION THREAD

**Discussion by Pedder007** [emsesp/EMS-ESP32#1087] [Q&A]
Hi all, I'm right fresh on my way with EMS and BBQKees and at least with our only 2 1/2 months old Bosch Compress 7800iLW (12kW) geothermic heating pump :-)
I already set up all to get the data points from the machine into ioBroker, but I fear, that I'm not able to see the really important ones, to at least stear the two heating circels which are attached to the heating. So I fear I need some guidance to get the right things in my hands :-)
Here are some more info about my technical setup:
Heating: Bosch CS7800iLW (12kW)
Steering module in the heating: UI800
Cable attached Bosch Gateway: K30RF (-> WLAN to Internet, at least not ping-able internally, but only shown in my network mgmt)
RC attached thermostat/steering: CR20RF (for the floor heating area)

BBQKees: v3.5.0b7
My setup was detected by it as following:
Heating (Boiler) detected as: Enviline/Compress 6000AW/Hybrid 7000iAW/SupraEco/Geo 5xx, which doesn't fit to my available one!?
Mixer (for the floor heating): MM100 which is correctly detected
2 Heatpump Modules without any data points
2 further Unknowns: w/o any data points
1 Sensor: also without any data points
![image](https://user-images.githubusercontent.com/44653371/221909666-9f765e0a-1ec5-4d95-ad49-98de5bdf18c6.png)

What I'm now really missing here, amongst the 85 available data points, are the parameters for the chosen room temperatures of the two heating circles and also the data from the thermostat.

Can someone guide me how to tackle this? I already spent some investigations yesterday, but in the end got lost in the tremendous amount of information here.

**Comment by proddy** [comment]
let's start by adding those two missing devices. Chances are one is the CR20. Click on the (i) on the right of the table row and send back the product id and device id. 

https://emsesp.github.io/docs/Troubleshooting/#not-all-ems-devices-are-recognized-or-data-is-missing

**Reply by proddy** [reply]
Also update the to latest version 3.5.0 which has more support for your device. The 3.5.0b7 you are using is from last October and we've been dropping in new stuff weekly since then.

**Reply by Pedder007** [reply]
Hi @proddy , thanks for your quick reply and your guidance:
The 'i's shows as following:
Heatpump: Prod ID: 252 Device ID: 0x48 Vers. 06.02
Heatpump: Prod ID: 252 Device ID: 0x50 Vers. 06.02
Unknown: Prod ID: 253 Device ID: 0x10 Vers. 47.06
Unknown: Prod ID: 249 Device ID: 0x39 Vers. 16.01
When clicking on the '+' in the sensor's row I get this input window:
![image](https://user-images.githubusercontent.com/44653371/221961079-abae178f-4312-446d-814f-be60fb61cf54.png)
Because of the contained warning, I didn't test around here yet :-)

I will now also try to get the update done and come back with the result.

**Comment by Pedder007** [comment]
Wow, that looks other :-)
![image](https://user-images.githubusercontent.com/44653371/221965956-040caadb-da85-4cde-8a61-533277503ef1.png)

Edit: Just quickly ran through the data points and all I need seems to be there! (in the Thermostat Rego 3000, which looks like the complete 'content' of the UI800)
Also setting them works, many many thanks!!! :-)

What just pops up in my mind: Does the machine also somewhere measure the pressure within the heating circles?
This would be good to have a remote monitoring on this, as esp. in radiator circles the pressures often varies and I already had to re-fill this circle 2-3 times since the pump is running here. For sure as there were rests of O2 in the pipes, but we also had it with our former oil-heating, that I had to re-fill 1-2 times per year. I found such a line in the boiler data points (directly below the for- and back-temp of the radiator circle), but it shows '0', whereas the system currently has ~1,3bar.
